MMDevice/Property.h

```diff
@@ -27,9 +27,64 @@
 #include <map>
 #include <string>
 #include <vector>
+#include <cmath>
 
 namespace MM {
 
+// Standard Properties
+const char* const g_KeywordStandardPropertyPrefix = "api//";
+
+// Define NaN for use in property definitions
+const double PropertyLimitUndefined = std::numeric_limits<double>::quiet_NaN();
+
+// Standard property metadata structure
+class StandardProperty {
```

Comment:
I think this can be a plain `struct` with all public fields. It clarifies that this is a pure-data object. There are no checks on the constructor arguments currently and `CreateStandardProperty` can perform checks when/if necessary.

Also eventually in C++20 we'll be able to use [designated initializers](https://en.cppreference.com/w/cpp/language/aggregate_initialization#Designated_initializers).